src/libs/state.js

```diff
@@ -75,6 +75,10 @@ export const snapshotStore = Utils.atom();
 /** @type {Utils.Atom<any[]>} */
 export const dataCatalogStore = Utils.atom([]);
 
+export const datasetBuilderCohorts = Utils.atom([]);
```

Comment:
While we have historically put all of our stores in `src/libs`, with a move towards more code ownership and organization, how would you feel about putting our state management files under our path?

Reply:
I guess, actually, since these aren't actually persisted anywhere in localstorage, it is probably fine to have two instances with the same name. I will localize them!